Add link to lima xbar plugin (#256)
lima-vm/lima: README.md

## Adopters

- [Colima](https://github.com/abiosoft/colima): Docker (and Kubernetes) on macOS with minimal setup

> - [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - Plugin for [xbar](https://xbarapp.com/) to create a Lima menu in your macOS menubar to start/stop your Lima VMs and see their running status.

Reviewer: Seems slightly verbose compared to the other entries; it is kind of implied that this is about Lima. 😄 

```suggestion
- [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - [xbar](https://xbarapp.com/) plugin to start/stop VMs from the menu bar and see their running status.
```

On second thoughts, this is just nit-picking, and the existing entry is fine too.

> - [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop

Reviewer 2: I don’t think the list has to be alphabetically sorted.

Also, could you squash commits?

Reviewer: > I don’t think the list has to be alphabetically sorted.

You'll need to let @unixorn know which order you prefer. Is it by original release date or something else?

> Also, could you squash commits?

Yes, the change done via the Github website is also missing the DCO line.

Author: Squashed the commits and signed the new squashed one.

I'm happy to revert the sort to the original order. It doesn't have to be alphabetical, I just feel that makes it easier to find things - not an issue now with three items, but for the future.
